types/react-loadable/test/webpack.ts

```diff
@@ -0,0 +1,25 @@
+import * as webpack from 'webpack';
+import { ReactLoadablePlugin, getBundles } from 'react-loadable/webpack';
```

Comment:
These also should be commonjs imports

```ts
import webpack = require('webpack');
import { ReactLoadablePlugin, getBundles } = require('react-loadable/webpack');
```

Reply:
Cool with the lack of destructuring? Doesn't seem possible.

Reply:
Sure, probably better that way as well.